debian/patches/debian/Don-t-run-pylint-at-build-time.patch

```diff
@@ -0,0 +1,23 @@
+From: Simon McVittie <smcv@debian.org>
+Date: Mon, 2 Jul 2018 18:41:57 +0100
+Subject: Don't run pylint at build time
```

Comment:
If you're still wondering: this patch is the reason that the pylint build failure went away once the source and packaging branches had matching names.

This patch should not be here. It's not made absolutely explicit on https://phabricator.endlessm.com/w/howto/rebase_debian_packages/ but the point of this instruction:

> 5. Inside the `debian/patches/` directory, set aside the patches that we want to keep (they'll be applied later).

is that we do not typically keep patches in `debian/patches/` on the packaging branch; we apply them to our code branch. So the packaging branch should not contain a `debian/patches` directory, and (if we want to keep this patch) it should be applied on the code branch.

Reply:
Oh, okay. Yes, the docs doesn't clarify it.

Hence, I suppose [this](https://github.com/endlessm/totem/pull/8/commits/90f789011fb71719eef9f0a0ca150e06898cafbf) and [this](https://github.com/endlessm/totem/pull/8/commits/6c864644e4e82d333d1e912a18612573e974f8e9) are patches from `3-26` packaging branch that are applied to code branch. And I have rebased them over again. Hence, those needs to drop right? 

I will apply `debian/patches/debian/Don-t-run-pylint-at-build-time.patch` patch to the code branch instead. Thanks for taking a look!

Reply:
Those 2 look like Endless-specific changes, not patches inherited from Debian – why would they need to be dropped? They could probably do with better commit messages. Perhaps you could do some Phabricator archaeology to work out why they were added, and (in the case of the first one) whether they could be upstreamed.

Reply:
heh right. Another **fast** reading fiasco. Just saw the `.patch` extension and assumed that they are applied on the code from previous debian packaging branch.

(My brain is not helping today :/ )